jobs/FreeBSD-head-amd64-test_ltp/meta/run.sh

```diff
@@ -12,6 +12,12 @@ for i in proc sys tmp; do
 done
 service linux start
 
+# runltp creates nobody, bin, and daemon users, but not root
+echo 'root:x:0:0:root::' > /etc/passwd
+
+# runltp creates nobody, bin, daemon, sys, and users groups, but not root
+echo 'root:x:0:' > /etc/group
```

Comment:
Are those the right destination? It looks modifying the system's `passwd` and `group` files. There is no `root` but why don't we have `root` user in test env? The syntax here looks like Linux's and should we modify `passwd` instead of `shadow`?

Reply:
Sigh, you're right.  Should be fixed now.